hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1220,14 +1220,14 @@
     {
       name: chip_sw_aes_enc
       uvm_test_seq: chip_sw_base_vseq
-      sw_images: ["//sw/device/tests:aes_smoketest:1"]
+      sw_images: ["//sw/device/tests:aes_smoketest:1:new_rules"]
```

Comment:
By the way, it seems there are some other "sw_images" missing the new_rules suffix.

Reply:
Can you point out where I've missed them?  I can't seem to find any with `grep`.